app/CocoaPods/CPDocumentController.swift

```diff
@@ -0,0 +1,35 @@
+import Cocoa
+
+@objc class CPDocumentController: NSDocumentController {
```

Comment:
Forgot about `NS*` objects inheriting that property. Removed the `@objc` and added a comment describing the purpose of the class.